dropwizard-jersey/src/main/java/io/dropwizard/jersey/validation/JerseyParameterNameProvider.java

```diff
@@ -0,0 +1,65 @@
+package io.dropwizard.jersey.validation;
+
+import org.hibernate.validator.parameternameprovider.ReflectionParameterNameProvider;
+
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.HeaderParam;
+import javax.ws.rs.CookieParam;
+import javax.ws.rs.FormParam;
+import javax.ws.rs.MatrixParam;
+import javax.ws.rs.core.Context;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Method;
+import java.lang.reflect.Parameter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class JerseyParameterNameProvider extends ReflectionParameterNameProvider {
+
+    @Override
+    public List<String> getParameterNames(Constructor<?> constructor) {
```

Comment:
Does this method need to be included? Looks like you're just delegating to `super`.